.github/workflows/greetings.yml

```diff
@@ -0,0 +1,25 @@
+permissions:
+  issues: write
+  pull-requests: write
+
+name: Greetings
+
+on:
+  fork:
+  push:
+    branches: [main]
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: EddieHubCommunity/gh-action-community/src/welcome@bf67289baedda0b13e216f7ca14e733eab899963
```

Comment:
Hi @CBID2, thanks for implementing this change. 

Question: where did you find this commit SHA? 
I do not see it in the main branch:
https://github.com/EddieHubCommunity/gh-action-community/commits/main
And this URL doesn't work:
https://github.com/EddieHubCommunity/gh-action-community/tree/bf67289baedda0b13e216f7ca14e733eab899963

Reply:
Hi @saragerion. Sorry for the long wait. I found a new greeting that they use: https://github.com/EddieHubCommunity/gh-action-community/blob/main/.github/workflows/community.yml
Here's the latest commit: https://github.com/EddieHubCommunity/gh-action-community/commit/92d4d80e6e42b2dac72e7c4c5102dc8a47ab019b